tools/rosconsole/src/rosconsole/rosconsole.cpp

```diff
@@ -425,6 +435,25 @@ void initialize()
     backend::function_notifyLoggerLevelsChanged = notifyLoggerLevelsChanged;
     backend::function_print = _print;
 
+    char* line_buffered = NULL;
+#ifdef _MSC_VER
+    _dupenv_s(&line_buffered, NULL, "ROSCONSOLE_STDOUT_LINE_BUFFERED");
```

Comment:
Actually please use `get_environment_variable()` which encapsulates the logic already: https://github.com/ros/roscpp_core/blob/5820d0672cb9d5fcd1c8cb4a159d04ff20115ff3/cpp_common/include/ros/platform.h#L63